common/static/coffee/src/discussion/views/discussion_thread_view.coffee

```diff
@@ -20,6 +20,7 @@ if Backbone?
       super()
       @mode = options.mode or "inline"  # allowed values are "tab" or "inline"
       @context = options.context or "course"  # allowed values are "course" or "standalone"
+      @course_settings = options.course_settings
```

Comment:
@ehteshamkafeel we can clean this later because changing this will take another hour to wait for tests.